apps/notifications/_handlers/messages/message-creation.handler.ts

```diff
@@ -0,0 +1,61 @@
+import type { Context } from '@azure/functions';
+import { NotificationCategoryEnum, ServiceRoleEnum, type NotifierTypeEnum } from '@notifications/shared/enums';
+import type { DomainContextType, NotifierTemplatesType } from '@notifications/shared/types';
+import { HandlersHelper } from '../../_helpers/handlers.helper';
+import { threadUrl } from '../../_helpers/url.helper';
+import { BaseHandler } from '../base.handler';
+
+export class MessageCreationHandler extends BaseHandler<
+  NotifierTypeEnum.THREAD_MESSAGE_CREATION,
+  'ME03_THREAD_MESSAGE_CREATION'
+> {
+  constructor(
+    requestUser: DomainContextType,
+    data: NotifierTemplatesType[NotifierTypeEnum.THREAD_MESSAGE_CREATION],
+    azureContext: Context
+  ) {
+    super(requestUser, data, azureContext);
+  }
+
+  async run(): Promise<this> {
+    const innovation = await this.recipientsService.innovationInfo(this.inputData.innovationId);
+    const recipients = await this.recipientsService.threadFollowerRecipients(this.inputData.threadId);
+
+    const recipientsByRole = this.recipientsService.getRecipientsByRole(recipients);
+
+    const senderName = await this.getRequestUserName();
+    const displayTag = HandlersHelper.getNotificationDisplayTag(this.requestUser.currentRole.role, {
+      unitName: this.requestUser.organisation?.organisationUnit?.name
+    });
+
+    for (const [role, roleRecipients] of Object.entries(recipientsByRole)) {
+      this.addEmails('ME03_THREAD_MESSAGE_CREATION', roleRecipients, {
+        notificationPreferenceType: NotificationCategoryEnum.MESSAGE,
+        params: {
+          innovation_name: innovation.name,
+          sender: `${senderName} (${displayTag})`,
+          thread_url: threadUrl(role as ServiceRoleEnum, innovation.id, this.inputData.threadId)
+        },
+        options: { includeSelf: false }
```

Comment:
this is the default. Can remove here and in the tests

Reply:
Changed, thanks!